tcbuilder/backend/combine.py

```diff
@@ -184,16 +188,13 @@ def combine_single_tezi_image(bundle_dir, files_to_add, output_dir, tezi_props):
     return update_tezi_files(output_dir, tezi_props, files_to_add)
 
 
-def check_combine_files(bundle_dir, is_for_raw_image=False):
+def check_combine_files(bundle_dir):
 
     files_to_add = []
     if bundle_dir is not None:
-        if not is_for_raw_image:
-            files_to_add = DOCKER_FILES_TO_ADD
-        else:
-            files_to_add = DOCKER_FILES_TO_ADD_TO_RAW
-
+        files_to_add = DOCKER_FILES_TO_ADD
```

Comment:
@lucas-akira I see here you're already tackling the issue I raised before about having two versions of this variable. So, please ignore that previous comment.

However, here I'd suggest copying the global variable rather than just referencing it because now we're making changes to it which wasn't true before. Changing global variables makes the code harder to reuse. For example this function would return a different result if called twice.

Reply:
Yes, the data is expected to be 1-level deep always.

Reply:
Alright, should be fixed now. Thanks!